website/app/templates/engineering.hbs

```diff
@@ -355,29 +360,19 @@
     React/SVG</h3>
 
   <p>
-    Single icons can be imported and used directly as SVG files using the
-    <a
-      href="https://react-components.vercel.app/components/inlinesvg"
-      target="_blank"
-      rel="noopener noreferrer"
-    >&lt;InlineSvg&gt;</a>
-    provided by the
-    <a
-      href="https://github.com/hashicorp/react-components"
-      target="_blank"
-      rel="noopener noreferrer"
-    >@hashicorp/react-components</a>
-    library:
+    Single icons can be also imported and used directly as standalone React/SVG components:
     {{! template-lint-disable 'no-potential-path-strings' }}
     {{! prettier-ignore-start }}
     <CodeBlock
       @language="js"
       @code="
-          // import the React file (TypeScript)
-          const FlightArrowRight = require('@hashicorp/flight-icons/svg-react/arrow-right-24');
+          // import the React/TypeScript file (using 'require')
+          const IconArrowRight24 = require('@hashicorp/flight-icons/svg-react/arrow-right-24?include');
+          // or import the React/TypeScript file (using 'import')
+          import { IconArrowRight24 } from '@hashicorp/flight-icons/svg/arrow-right-24.svg?include';
```

Comment:
Noticed a few things here:

- the `?include` and `.svg?include` shouldn't be in the require/important statements
- I needed to put curly braces around `IconArrowRight24` for the `require` statement to work
- the import statement needs to be updated to be from `@hashicorp/flight-icons/svg-react` instead of `@hashicorp/flight-icons/svg`

```suggestion
          // import the React/TypeScript file (using 'require')
          const { IconArrowRight24 } = require('@hashicorp/flight-icons/svg-react/arrow-right-24');
          // or import the React/TypeScript file (using 'import')
          import { IconArrowRight24 } from '@hashicorp/flight-icons/svg-react/arrow-right-24';
```

Reply:
@ashleemboyer thanks for the suggestion/fix.

regarding the `?include` I followed in the import/require statements in the DevPortal codebase:
https://github.com/hashicorp/dev-portal/blob/d82ace559de1226bf3d75c72197af6de03b5a1ed/src/components/_proxied-dot-io/waypoint/homepage/sections/how-it-works/build-and-deploy/index.tsx#L76
https://github.com/hashicorp/dev-portal/blob/1a000fa43c9fdeed2e90f1a579b0c251c7663246/src/components/navigation-header/index.tsx#L3

Reply:
No problem! 😊 So the `?include` in the import statement works when we're importing an svg file and using the `InlineSvg` component, but not when we're importing React components.

Reply:
Ohhhh! I see :) good to know!